library/core/src/fmt/num.rs

```diff
@@ -1,6 +1,6 @@
 //! Integer and floating-point number formatting
 
-use crate::mem::MaybeUninit;
+use crate::mem::{MaybeUninit, size_of};
```

Comment:
This failed CI because `size_of` is only used in these functions, which are gated behind `#[cfg(not(feature = "optimize_for_size"))]`. But `size_of` is now in the prelude so you can just remove this import.

Reply:
How "… you can just remove this import"?